components/profile/CollectionFilter.vue

```diff
@@ -0,0 +1,168 @@
+        no-shadow
+        rounded
+        label="Collections"
+        :icon="active ? 'chevron-up' : 'chevron-down'" />
+
+      <ActiveCount :count="checked.length" position="top-right" />
+    </template>
+
+    <div class="py-5 dropdown-width">
+      <NeoDropdownItem
+        v-for="collection in collections"
+        :key="collection.id"
+        class="is-flex no-border is-justify-content-center is-align-items-center"
+        aria-role="listitem"
+        :value="collection.id">
+        <NeoCheckbox
+          :model-value="isSelected(collection)"
+          class="pointer-events-none" />
+        <div
+          class="is-flex is-align-items-center filter-container is-flex-grow-1 min-width-0">
+          <img
+            :src="sanitizeIpfsUrl(collection.meta.image)"
+            class="image is-32x32 is-flex-shrink-0 border mr-2"
+            :alt="collection.name || collection.id" />
+          <div
+            class="is-flex is-flex-direction-column is-flex-grow-1 min-width-0">
+            <div class="is-ellipsis">
+              {{ collection.name || collection.id }}
+            </div>
+
+            <div
+              class="is-flex is-justify-content-space-between is-size-7 has-text-grey">
+              <div>{{ $t('search.owners') }}: {{ collection.owners }}</div>
+            </div>
+          </div>
+
+          <div class="rounded ml-5 px-3 k-grey-light">
+            {{ collection.owned }}
+          </div>
+        </div>
+      </NeoDropdownItem>
+    </div>
+  </NeoDropdown>
+</template>
+
+<script setup lang="ts">
+import {
+  NeoButton,
+  NeoCheckbox,
+  NeoDropdown,
+  NeoDropdownItem,
+} from '@kodadot1/brick'
+import collectionListWithSearch from '@/queries/subsquid/general/collectionListWithSearch.graphql'
+import ActiveCount from '../explore/ActiveCount.vue'
+import { CollectionEntityMinimal } from '@/components/collection/utils/types'
+import { getDenyList } from '@/utils/prefix'
+import isEqual from 'lodash/isEqual'
+
+type Collection = CollectionEntityMinimal & {
+  owners: number
+  owned: number
+}
+
+const props = defineProps<{
+  id: string
+  search: Record<string, string | number>
+  tabKey: string
+}>()
+
+const checked = ref<string[]>([])
+
+const { urlPrefix, client } = usePrefix()
+const { replaceUrl } = useReplaceUrl()
+
+const collections = ref<Collection[]>([])
+
+const nonCollectionSearchParams = computed(() => {
+  const search = { ...props.search }
+  delete search.collection
+  return search
+})
+
+useLazyAsyncData('profileCollections', async () => {
+  await getProfileCollections()
+})
+
+const getProfileCollections = async () => {
+  const collectionSearch = {
+    nfts_some: nonCollectionSearchParams.value,
+  }
+
+  const { data } = await useAsyncQuery({
+    query: collectionListWithSearch,
+    variables: {
+      search: [collectionSearch],
+      denyList: getDenyList(urlPrefix.value),
+      first: 100,
+      offset: 0,
+    },
+    clientId: client.value,
+  })
```

Comment:
Only the first 100 collections would be loaded here. What if we remove the limit?

Reply:
OK. As it's used only for one user's profile, we can expand the limit to 200 or so if need.